src/System.Private.CoreLib/src/System/Threading/Thread.CoreRT.cs

```diff
@@ -137,6 +137,12 @@ private bool HasStarted()
             return !GetThreadStateBit(ThreadState.Unstarted);
         }
 
+        internal SynchronizationContext SynchronizationContext
```

Comment:
It would be better to change the place that uses this to use `_synchronizationContext` directly instead. (It is the style we use for these everywhere else.)